.changesets/feat_renee_api_schema.md

```diff
@@ -0,0 +1,11 @@
+## Experimental support for API Schema generation in Rust
+
+As part of the process of replacing JavaScript code in the Router with more reliable and performant Rust implementations, the API schema can now be generated in Rust. The new `experimental_api_schema_generation_mode` option controls which implementation is used.
+
+```yaml {4,8} title="router.yaml"
```

Comment:
TIL you can do this in markdown formatting?? very cool

Reply:
It's just copy pasted from the docs code which does something special, I don't think it does anything in github markdown :)